.gitignore

```diff
@@ -10,6 +10,7 @@ public/wp-config.php
 public/wp-content/uploads
 
 #gitignore for theme folder
+!public/wp-content/themes/Toolbox-2020/src/scss/assets/bootstrap/vendor
```

Comment:
Why is this in boilerplate? @kuldeep3

Reply:
@rathorevaibhav Changing the theme name to the boilerplate theme name.